### ISSUE TITLE
[BUG] False error when trying to order on invalid API credentials

### ISSUE THREAD

**Issue by vandijkstef** [tig-nl/gls-magento2#20]
**To Reproduce**
Try to order with invalid API credentials.
When the API credentials are invalid, the front-end/checkout page is not showing the sub-options (like ship or pick-up point).

**Expected result**
A different error stating either 'Something went wrong, try again later' or 'Could not reach GLS API' instead of putting blame to the front-end users data entry. Additionally, nothing was found in the various './var/log/*' log-files leaving the developer or site-owner completely guessing.

The error is blocking the order, which might even be harmful for our business should it happen again. Accepting the order as 'default shipment' and silently fail on the pick-ups points would be preferred. This is probably connected to #17 and #19 

**Actual result**
The error message tells me to 'check my address input' while there was a correct and valid address in the fields.

**Workaround**
Simply put in the correct API credentials. I have not changed the credentials, and I am unsure why and how the system 'forgot' the actual credentials. It might have possibly been wrongly saved changing other configuration options, since I have been visiting the config page.

Please note that due to the false error statement, I spend about 2 hours reviewing changes in XML files, rebuilding DI, refreshing caches, drinking insane amounts of coffee and pulling my hairs out.

** Please complete the following information**
- Magento version: 2.3.7
- PHP version: 7.3.X
- TIG GLS version: 1.3.2

**Comment by tig-jeffreybranderhorst** [comment]
Hi @vandijkstef ,

Thank you for contacting us and for giving us an explanation. We will take a good look at all of it with the team and fix the issues!

Have a great day,
Jeffrey
